src/assets/src/components/meetingTables.tsx

```diff
@@ -98,6 +98,15 @@ const InvalidMeeting = (props: MeetingEditorProps) => {
 
 }
 
+// format the time in seconds to hours, minutes, and seconds
+function formatTimeInSeconds(timeInSeconds: number): string {
+    const hours = Math.floor(timeInSeconds / 3600);
+    const minutes = Math.floor((timeInSeconds % 3600) / 60);
+    const seconds = timeInSeconds % 60;
+
+    return `${hours} hours ${minutes} minutes ${seconds} seconds`;
```

Comment:
Probably _minor_ but having the (s) in the plural in a parenthesis would probably be more appropriate.

`    return `${hours} hour(s) ${minutes} minute(s) ${seconds} second(s)`;
`
It could probably handle the special case in logic but I don't know if that's necessary. 

This was GPT generated

```
    function formatUnit(value: number, unit: string): string {
        return `${value} ${unit}${value !== 1 ? 's' : ''}`;
    }

    return `${formatUnit(hours, 'hour')} ` +
           `${formatUnit(minutes, 'minute')} ` +
           `${formatUnit(seconds, 'second')}`;
```

Reply:
Or use an ISO 8601 kind of format…

```
return `${hours}h${minutes}m${seconds}s`
```

Or simple timestamp style.  I think people are used to seeing this format in video timestamps…

```
return `${hours}:${minutes}:${seconds} (h:m:s)`
```

Could add a "legend" just in case.

Reply:
I have implemented the string change with the unit pluralized if the value is greater than 1

Reply:
and also added the refresh in every 5 seconds

Reply:
Not to keep nitpicking on this but I kind of like Lance's suggestion. Maybe compact it down to just '#h #m #s' and then we don't have to worry about the plural? Maybe Hema has a suggestion?

I think it's fine but having '0 hour' I think i incorrect and that needs to be plural or dropped like

`if (value == 0) return '';`